tools/integration-test/src/tests/execute_schedule.rs

```diff
@@ -0,0 +1,82 @@
+//! This test ensures that the `RelayPath::execute_schedule` method does not
+//! drop any pending scheduled operational data when a prior transaction fails
+//! to send. Subsequent pieces of operational data that were scheduled should
+//! be re-queued and not dropped.
+//!
+//! In order to test this behavior, the test manually relays at least 2 IBC transfers
+//! from chain A to chain B. Chain B is then shut down in order to force the transactions
+//! to be queued up again for re-submission. It is expected that the first transfer
+//! be dropped and not be present in the pending queue, but all of the subsequent
```

Comment:
Nit: these are not transactions, but operational data (i.e., batches or messages waiting to be relayed).